### ISSUE TITLE
Suspicious Train/Val loss behaviours?

### ISSUE THREAD

**Issue by Jumperkables** [Visual-modelling/cnn_visual_modelling#4]
Train and validation losses tend to behave similary very late into training. Keep an eye
